modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,441 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/manage-objects-with-rosa.adoc
+
+[id="rosa-create-objects{context}"]
+= Create objects
+
+This section describes the `create` commands for clusters and resources.
+
+[id="rosa-create-admin_{context}"]
+== create admin
+
+Creates a cluster administrator with an automatically generated password that can log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create admin --cluster=<clusterName>
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+Create a cluster administrator that can log into a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create admin --cluster-name=mycluster
+----
+
+[id="rosa-create-cluster_{context}"]
+== create cluster
+
+Creates a new cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create cluster --cluster=<clusterName> [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster. This is used when generating a sub-domain for your cluster on openshiftapps.com.
+
+|--channel-group
+|The channel group (string) is the name of the group where this image belongs, for example _stable_ or _fast_. Default: stable
+
+|--compute-machine-type
+|The instance type (string) for the compute nodes. Determines the amount of memory and vCPU allocated to each compute node.
+
+|--compute-nodes
+|The number (integer) of worker nodes to provision per zone. Single zone clusters require at least 2 nodes. Multi-zone clusters require at least 3 nodes. Default: 2
+
+|--disable-scp-checks
+|Indicates whether cloud permission checks are disabled when attempting the installation of the cluster.
+
+|--dry-run
+|Simulates creating the cluster.
+
+|--host-prefix
+|The subnet prefix length (integer) to assign to each individual node. For example, if host prefix is set to _23_, then each node is assigned a /23 subnet out of the given CIDR.
+
+|--machine-cidr
+|Block of IP addresses (ipNet) used by OpenShift while installing the cluster. Example: 10.0.0.0/16
```

Comment:
We usually don't write just "OpenShift". Instead we use {product-title}, though I'm not sure if that's correct for ROSA or not.

Reply:
Can't be changed at this time due unless something can be changed in how we use the attributes file.